mod/dfrn_request.php

```diff
@@ -514,6 +514,11 @@ function dfrn_request_post(App $a) {
 					return; // NOTREACHED
 				}
 
+				if (blocked_url($url)) {
+					notice( t('Blocked domain') . EOL);
```

Comment:
Yes, we need to make this obvious. At least in the docs, that the reason for the block will be listed there prominentely.